docs/getting-started/creating-your-first-bot.mdx

```diff
@@ -200,6 +200,8 @@ decorator to define a slash command. We specify the `name` and `description` arg
 specified, the name of the slash command would be the function name and the command description would
 be empty.
 
+Optional: We typehint [': discord.ApplicationContext'](https://docs.pycord.dev/en/stable/api/application_commands.html#discord.ApplicationContext) to get access to autocompletion and docstrings.
```

Comment:
```suggestion
Optional: We type-hint [`discord.ApplicationContext`](https://docs.pycord.dev/en/stable/api/application_commands.html#discord.ApplicationContext). In many modern IDEs, we can use this to access autocompletion and docstrings. You can read more about type-hinting [here](https://docs.python.org/3/library/typing.html).
```

Reply:
I'd use:
[...] we use this to access [...].
Without "can", because it's already marked as optional.

Otherwise I like your version better. appears more professional.

Reply:
```suggestion
Optional: We type-hint the context parameter (`ctx`) as [`discord.ApplicationContext`](https://docs.pycord.dev/en/stable/api/application_commands.html#discord.ApplicationContext). In modern IDEs, type-hinting allows access to autocompletion and docstrings. You can read more about type-hinting [here](https://docs.python.org/3/library/typing.html).
```